warehouse/query-core/src/test/java/datawave/query/function/HitsAreAlwaysIncludedTest.java

```diff
@@ -260,8 +276,16 @@ public void testHitForIndexedQueryTerm() throws Exception {
         
         String queryString = "FOO_3_BAR == 'defg<cat>'";
         
-        Set<String> goodResults = Sets.newHashSet("FOO_1_BAR.FOO.3:good<cat>", "FOO_3_BAR.FOO.3:defg<cat>", "FOO_3.FOO.3.3:defg", "FOO_4.FOO.4.3:yes",
-                        "FOO_1.FOO.1.3:good");
+        //@formatter:off
+        Set<String> goodResults = Sets.newHashSet(
+                "FOO_1_BAR.FOO.3:good<cat>", "FOO_3_BAR.FOO.3:defg<cat>", "FOO_3.FOO.3.3:defg", "FOO_4.FOO.4.3:yes", "FOO_1.FOO.1.3:good",
+                "FOO_1.FOO.1.0:yawn",
+                "FOO_1_BAR.FOO.0:yawn<cat>", "FOO_1_BAR.FOO.1:yawn<cat>",
+                "FOO_1_BAR_1.FOO.0:2021-03-24T16:00:00.000Z",
+                "FOO_3.FOO.3.0:abcd",
+                "FOO_3_BAR.FOO.0:abcd<cat>",
+                "FOO_4.FOO.4.0:purr", "FOO_4.FOO.4.1:purr");
```

Comment:
Is there a rhyme or reason to some lines having multiple values, and others having single values?  If we're going to group by field (which I think was your intention) let's do that for all of the results, not just the new ones you added.

Reply:
The first line is the set of hits, and the other lines are the addition fields included beyond the hits.  I will try to document as such.

Reply:
thanks